File: app/src/main/java/io/github/jamilelima/pomofocus/Fragments/TasksDoneFragment.java
package io.github.jamilelima.pomofocus.Fragments;


import android.arch.persistence.room.Room;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import io.github.jamilelima.pomofocus.Adapters.TasksDoneAdapter;
import io.github.jamilelima.pomofocus.Adapters.TasksPendingAdapter;
import io.github.jamilelima.pomofocus.AppDatabase;
import io.github.jamilelima.pomofocus.Model.Task;
import io.github.jamilelima.pomofocus.R;
import java.util.List;


public class TasksDoneFragment extends Fragment {

    private RecyclerView tasksDoneRecyclerView;
    private TasksDoneAdapter tasksDoneAdapter;


    public TasksDoneFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
       View rootView = inflater.inflate(R.layout.fragment_tasks_done, container, false);

        AppDatabase db = Room
            .databaseBuilder(getActivity().getApplicationContext(), AppDatabase.class, "production").allowMainThreadQueries().build();

        List<Task> tasksList = db.taskDao().getDoneTasks();

        tasksDoneRecyclerView = rootView.findViewById(R.id.task_done_recyclerview);
        tasksDoneAdapter = new TasksDoneAdapter(getActivity(), tasksList);
        tasksDoneRecyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
        tasksDoneRecyclerView.setAdapter(tasksDoneAdapter);

       return rootView;
    }

}

File: app/src/main/java/io/github/jamilelima/pomofocus/fragments/TasksDoneFragment.kt
package io.github.jamilelima.pomofocus.fragments


import android.arch.persistence.room.Room
import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v7.widget.LinearLayoutManager
import android.support.v7.widget.RecyclerView
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import io.github.jamilelima.pomofocus.adapters.TasksDoneAdapter
import io.github.jamilelima.pomofocus.AppDatabase
import io.github.jamilelima.pomofocus.R
import kotlinx.android.synthetic.main.fragment_tasks_done.view.*


class TasksDoneFragment : Fragment() {

    private var tasksDoneRecyclerView: RecyclerView? = null
    private var tasksDoneAdapter: TasksDoneAdapter? = null


    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {

        val rootView = inflater.inflate(R.layout.fragment_tasks_done, container, false)

        val db = Room
                .databaseBuilder(activity!!.applicationContext, AppDatabase::class.java, "production").allowMainThreadQueries().build()

        val tasksList = db.taskDao().doneTasks

        tasksDoneRecyclerView = rootView.task_done_recyclerview
        tasksDoneAdapter = TasksDoneAdapter(activity!!, tasksList)
        tasksDoneRecyclerView?.layoutManager = LinearLayoutManager(activity)
        tasksDoneRecyclerView?.adapter = tasksDoneAdapter

        return rootView
    }

}
